src/celeritas/em/distribution/UrbanMscScatter.hh

```diff
@@ -544,15 +550,13 @@ CELER_FUNCTION Real3 UrbanMscScatter::sample_displacement(Engine&   rng,
 /*!
  * Scale displacement and correct near the boundary.
  */
-CELER_FUNCTION real_type UrbanMscScatter::calc_displacement_scaling(
-    const Real3& displacement, real_type rmax2)
+CELER_FUNCTION real_type
+UrbanMscScatter::calc_displacement_length(real_type rmax2)
 {
     CELER_EXPECT(rmax2 >= 0);
-    // Scale with the lateral arm
-    real_type multiplier = real_type(0.73) * std::sqrt(rmax2);
 
+    real_type rho = real_type(0.73) * std::sqrt(rmax2);
```

Comment:
Can you take this opportunity to add a short description about `0.73` that we talked about (i.e., see the description for u [here](https://geant4.kek.jp/lxr/source/processes/electromagnetic/standard/src/G4UrbanMscModel.cc?v=10.5#L1137)) and put something like, "the average value of a simple distribution of r=u*rmax based on the single scattering result"?

Reply:
This MR was merged a few days ago, but I'll update it in my current branch :)